building/building-archlinux-template.md

```diff
@@ -333,15 +334,15 @@ redirect_from:
 *   **Transfer the install-templates.sh script file into Dom0**
   *Note: as there is not a typical file transfer method for Dom0, for security reasons, this less than simple transfer function has to be used*
 
-    *   Switch to Domo and open a terminal window.
+    *   Switch to Dom0 and open a terminal window.
 
     **Note:** Take care when entering these cmd strings.  They are very long and have a number of characters that are easy to mix '**-**' vs '**.**' '**<u>T</u>emplates** (correct) vs **<u>t</u>emplates** (wrong) or **Template_**'(also wrong)  This script will also take care of transfering the actual template.rpm to Dom0 as well.
 
-       *   **$ qvm-run --pass-io development 'cat /home/user/qubes-builder/qubes-src/linux-template-builder/rpm/install-templates.sh' > install-templates.sh**
+       `$ qvm-run --pass-io development 'cat /home/user/qubes-builder/qubes-src/linux-template-builder/rpm/install-templates.sh' > install-templates.sh`
        
-       *   **$ chmod +x install-templates.sh**
+       `$ chmod +x install-templates.sh`
        
-       *   **$ ./install-templates.sh**
+       `$ ./install-templates.sh`
```

Comment:
For example, this series of commands should probably be a single code block.

Reply:
I liked the separation but it did not look as consistent as it should have so I've taken you advice.